common/src/main/java/com/codeforcommunity/email/EmailOperations.java

```diff
@@ -179,20 +179,6 @@ public void sendEmailToOneRecipient(
     this.sendEmail(email, subject);
   }
 
-  /**
-   * Send an email with the given subject and body to the user with the given name at the given
-   * email.
-   */
-  public void sendEmailToOneRecipient(
-      String sendToName, String sendToEmail, String subject, String emailBody) {
-    if (!shouldSendEmails) {
-      return;
-    }
-    logger.info(String.format("Sending email with subject `%s`", subject));
-    Email email = buildEmailSingleRecipient(sendToName, sendToEmail, subject, emailBody, new ArrayList<>());
-    this.sendEmail(email, subject);
-  }
-
```

Comment:
Up to you if you want to make any more changes here - for future reference, we should be able to do something like this method while avoiding duplicating code by just having its body call the other version with an empty list, so something like this:

```
public void sendEmailToOneRecipient(String sendToName, String sendToEmail, String subject, String emailBody) {
  this.sendEmailToOneRecipient(sendToName, sendToEmail, subject, emailBody, new ArrayList<>());
}
```